error_handling.ipynb

```diff
@@ -28,7 +28,10 @@
     "    try:\n",
     "        map_obj = map(int, int_list)\n",
     "        int_list = list(map_obj)\n",
+    "        \n",
     "        sum = 0\n",
+    "        \n",
+    "        # sum the items in the intlist\n",
     "\n",
     "        for i in range(len(int_list)):\n",
```

Comment:
what about removing this loop. just :

`data = sum(int_list)'
`return data`

Reply:
yes, agree that